python/lsst/pex/config/configurableField.py

```diff
@@ -308,9 +324,9 @@ def __getOrMake(self, instance, at=None, label="default"):
             instance._storage[self.name] = value
         return value
 
-    def __get__(self, instance, owner=None, at=None, label="default"):
+    def __get__(self, instance, owner=None, at=None, label="default") -> ConfigurableInstance[FieldTypeVar]:
         if instance is None or not isinstance(instance, Config):
-            return self
+            return self  # type: ignore
```

Comment:
I think this ignore is only needed because you didn't mark the `__get__` as an overload this time (with an un-annotated or `Any`-annotated implementation).  I think that's the right fix.